src/wallet/wallet2.cpp

```diff
@@ -12081,7 +12082,8 @@ bool wallet2::get_tx_key(const crypto::hash &txid, crypto::secret_key &tx_key, s
     crypto::hash tx_hash{};
     cryptonote::blobdata tx_data;
     crypto::hash tx_prefix_hash{};
-    bool ok = string_tools::parse_hexstr_to_binbuff(res.txs_as_hex.front(), tx_data);
+    auto res_tx = res.txs.front();
+    bool ok = string_tools::parse_hexstr_to_binbuff(*res_tx.pruned_as_hex + (res_tx.prunable_as_hex ? *res_tx.prunable_as_hex : ""s), tx_data);
```

Comment:
since this line is being modified anyway, should this be named something that makes a little more sense (was_parsed?)

Reply:
Also that `res_tx` was supposed to be a reference.  Hmm.

Reply:
lol, that's a pretty lucky find